adoc/reference-webui-systems.adoc

```diff
@@ -2644,6 +2644,8 @@ Commas are problematic because they are used as separator when two or more activ
 Selecting `{productname} Default` allows client systems to register with the {suse}-provided default channel that corresponds with their installed version of {sle}.
 You can also associate the key with a custom base channel.
 If a system using this key is not compatible with the selected channel, it will fall back to the {productname} default channel.
+. [guimenu]``Child Channels`` -- When the base channel is selected the list of available child channels will get fetched and display in real time below the base channel.
```

Comment:
Can we replace the double hyphen with an actual em-dash? Also, it should probably be unspaced. Might need to check the style on that. Otherwise, colons might work better in this section?

Reply:
ok, entered as https://github.com/SUSE/spacewalk/issues/6119 --FTM, we keep it as-is.